GeneralStateTests/stMemoryTest/codeCopyOffset.json

```diff
@@ -45,7 +89,7 @@
             },
             "0xeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee" : {
                 "balance" : "0x0de0b6b3a7640000",
-                "code" : "0x7fffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff600052601061ffff61010037600051600055",
+                "code" : "0x7fffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff600052601061ffff600039600051600055",
```

Comment:
I feel like a detective here.

Is this change intended?  Even if unintended, it's fine if it's harmless.

Reply:
No.  it was unintended.  But you're right, it shouldn't matter for this test case (as long as source offset > source size, and the amount of data being written > 0)